examples/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'java'
+}
+
```

Comment:
gradle wrapper executables are missing inside `examples` folder. You can copy/paste `gradlew`, `gradlew.bat` and `gradle` folder from parent folder.

Reply:
To verify that everything works please run
`cd examples`
`./gradlew clean build`

Reply:
Done